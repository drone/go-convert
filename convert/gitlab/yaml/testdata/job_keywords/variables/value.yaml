# https://docs.gitlab.com/ee/ci/yaml/#variablesvalue

variables:
  DEPLOY_ENVIRONMENT:
    value: "staging"
    description: "The deployment target. Change this variable to 'canary' or 'production' if needed."