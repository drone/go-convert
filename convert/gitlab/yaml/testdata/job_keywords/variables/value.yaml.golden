variables:
    DEPLOY_ENVIRONMENT:
        value: staging
        description: The deployment target. Change this variable to 'canary' or 'production' if needed.
        expand: true
